multimedia/templates/multimedia/base.html

```diff
@@ -0,0 +1 @@
+{% extends "base.html" %}
```

Comment:
I think we should split `multimedia`, `books`, `music`, `movies`, `applications` as separate django apps.